include/overlay097/box_pokemon_gba.h

```diff
@@ -0,0 +1,95 @@
+#ifndef POKEPLATINUM_BOX_POKEMON_GBA_H
+#define POKEPLATINUM_BOX_POKEMON_GBA_H
+
+typedef struct {
+    u16 species;
+    u16 heldItem;
+    u32 experience;
+    u8 ppBonuses;
+    u8 friendship;
+    u16 filler;
```

Comment:
this should be padding not filler, but should probably stay consistent with gen 3 repo names (i.e. this is just a note, don't change it_)

Reply:
I mean, you're right, it's padding, so the Gen 3 repos should rename it too.